src/components/OrderItem/OrderItem.jsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+
+export class OrderItem extends React.Component {
+  constructor(props) {
+    super(props)
+    this.action = this.action.bind(this);
+  }
+
+  action() {
+    this.props.removeFromOrder(this.props.index, this.props.price);
+  }
+
+  render() {
+    const { name, price, bread, isHot } = this.props;
+
+    return (
+      <li className="order-item">
+        <p className="order-item__name">
+          {name}
+          <span className="order-item__price">&pound;{price.toFixed(2)}</span>
+        </p>
+        {isHot && <small className="hot">HOT</small>}
+        <small>{bread !== "Standard" && bread}</small>
```

Comment:
This will render an empty `<small>` tag if the bread is standard. Might want to move the conditional outside of the tag 👍

Reply:
good spot man